ui/controls.js

```diff
@@ -675,11 +676,15 @@ shaka.ui.Controls.prototype.addControlsContainer_ = function() {
   this.videoContainer_.appendChild(this.controlsContainer_);
 
   this.eventManager_.listen(this.controlsContainer_, 'touchstart', (e) => {
-    this.onContainerTouch_(e);
+    if (!e.defaultPrevented) {
```

Comment:
This looks like it's probably a useful change independent of the Cast changes.  What's the use case?  Would you be willing to split that into a separate PR?